src/main/resources/application.properties

```diff
@@ -4,3 +4,5 @@ spring.datasource.password=wolox
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
 spring.jpa.hibernate.ddl-auto=update
 server.port=8081
+spring.security.user.name=user
```

Comment:
Just as a general comment for the future, be careful as to not upload credentials or sensitive info to github, in general you will use the IDE environment configuration and set up the credentials as secrets in the deployment